### ISSUE TITLE
My Teperature humidity sensor zigbee_zn373186_temphumid_sensor is not recognized 

### ISSUE THREAD

**Issue by romicaiarca** [make-all/tuya-local#1250]
My Teperature humidity sensor zigbee_zn373186_temphumid_sensor is not recognized. This workd fine few days ago.

**Describe the bug**
My Teperature humidity sensor zigbee_zn373186_temphumid_sensor is not recognized. This workd fine few days ago.

This is the output of localtuya wizard
```
{
            "active_time": 1670844095,
            "biz_type": 0,
            "category": "wsdcg",
            "create_time": 1670841274,
            "icon": "smart/icon/ay1562830083757AoBrF/cfb6e086fbcecef9711b728b85024b04.png",
            "id": "bf2...2gu",
            "ip": "",
            "lat": "45.6928",
            "local_key": "8d2...068",
            "lon": "27.1922",
            "model": "",
            "name": "THS",
            "node_id": "a4c...6c4",
            "online": false,
            "owner_id": "44...62",
            "product_id": "xr3htd96",
            "product_name": "Temperature Humidity Sensor",
            "status": [
                {
                    "code": "va_temperature",
                    "value": -94
                },
                {
                    "code": "va_humidity",
                    "value": 705
                },
                {
                    "code": "battery_percentage",
                    "value": 1
                }
            ],
            "sub": true,
            "time_zone": "+02:00",
            "uid": "eu16...zApB",
            "update_time": 1672349563,
            "uuid": "a4c...6c4"
        },
```
**To Reproduce**
Add the sensor into HA

**Expected behavior**
I expect to get in the recognized  list the definision from zigbee_zn373186_temphumid_sensor

**Comment by make-all** [comment]
Wrong repository for report

**Comment by romicaiarca** [comment]
This wasn't the wrong repo report. I just tried to add in HA via tuya-local my sensors. I've added the output for sensors from localtuya, to give you an ideea about how you handle the recognition and that my sensor should work fine with de config from the file zigbee_zn373186_temphumid_sensor.yaml.

Anyways, the solution was to restart few times the sensors (remove battery, wait for few minutes and add back the batteries, and the add again in HA. This was the device was properly recognized and works fine now.
